torch/_dispatch/python.py

```diff
@@ -1,5 +1,9 @@
 import torch._C
 from contextlib import contextmanager
+import unittest.mock
```

Comment:
As this is not part of `torch.testing.` folder I wonder if it would make more sense to move it under `enable_crossref_functionalize`
```suggestion
```

Reply:
I mean, I'm mostly using this as a convenient monkey patcher, it doesn't seem like a huge problem to import it at top level

Reply:
If this module will ever be imported by default, it could be a problem, when it end up in someones code as  `import torch.dispatch.python.mock`

Reply:
Surely you're joking, Mr. Shulga! One could similarly fret over every import statement in all of our PyTorch modules! Surely the cow has already escaped the barn on this one.